build/docker.mk

```diff
@@ -1,9 +1,15 @@
 #
 # Makefile fragment for Docker actions
 #
-DOCKER            ?= docker
 DOCKER_FILE       ?= build/package/Dockerfile
-DOCKER_IMAGE      ?= newrelic/k8s-operator:snapshot
+DOCKER_IMAGE      ?= newrelic/k8s-operator
+DOCKER_IMAGE_TAG  ?= snapshot
```

Comment:
Will `DOCKER_IMAGE_TAG ` get automatically updated with a semver version when we release?

Reply:
Go releaser doesn't use this make target, so it isn't in the general release path at all.  I pulled it out as a different var just so it could be overridden manually without modification to the make file.